tests/zwe-remote-integration/src/__tests__/unit/__configmgr__/tests/launchScript.cmgr.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+import * as config from '../bin/libs/config';
+import { _unit_test } from '../bin/commands/internal/start/prepare/index';
+import { assertEqualsStrict } from './common/assert';
+import * as common from '../bin/libs/common';
+
+const ZOWE_CONFIG = config.getZoweConfig();
+common.printMessage('Starting "getStartupCheckMode" test cases.');
```

Comment:
Not a problem now, but if we do more unit tests, there might be the functions of the same name or a function (like in this example) not defined in `libs`. We might start identifying like "/bin/command/internal/start/prepare -> getStartupCheckMode". Just for consideration.